watsonsdk/stt/STTConfiguration.m

```diff
@@ -102,6 +102,7 @@ - (NSString *)getStartMessage{
     [inputParameters setValue:self.interimResults forKey:@"interim_results"];
     [inputParameters setValue:self.continuous forKey:@"continuous"];
     [inputParameters setValue:self.inactivityTimeout forKey:@"inactivity_timeout"];
+    [inputParameters setValue:self.learningOptOut forKey:@"x-watson-learning-opt-out"];
```

Comment:
@mihui I was looking at the docs and they say query parameter:
http://www.ibm.com/watson/developercloud/doc/text-to-speech/websockets.shtml Am I looking at the correct place?

I checked in the demo app https://speech-to-text-demo.mybluemix.net/ in [index.js](https://speech-to-text-demo.mybluemix.net/js/index.js) and found that "X-Watson-Learning-Opt-Out" is not being set. It is a TODO.  So I am confused. How can I check the value was correctly set?